ql/math/randomnumbers/fractionalnoisersg.hpp

```diff
@@ -0,0 +1,149 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
+ Copyright (C) 2022 Chester Wong
+ Copyright (C) 2022 Wojciech Czernous
+
+ This file is part of QuantLib, a free-software/open-source library
+ for financial quantitative analysts and developers - http://quantlib.org/
+
+ QuantLib is free software: you can redistribute it and/or modify it
+ under the terms of the QuantLib license.  You should have received a
+ copy of the license along with this program; if not, please email
+ <quantlib-dev@lists.sf.net>. The license is also available online at
+ <http://quantlib.org/license.shtml>.
+
+ This program is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ FOR A PARTICULAR PURPOSE.  See the license for more details.
+*/
+
+/*! \file fractionalnoisersg.hpp
+    \brief Fractional Gaussian noise (sequence) generator
+*/
+
+#ifndef quantlib_fractional_gaussian_noise_rsg_hpp
+#define quantlib_fractional_gaussian_noise_rsg_hpp
+
+#include <ql/math/fastfouriertransform.hpp>
+#include <ql/math/matrix.hpp>
+#include <ql/methods/montecarlo/sample.hpp>
+#include <complex>
+
+using std::pow;
```

Comment:
`using` should not be used in header files, as it forces everybody that depends on this header to pull it into their project. In this case the best solution is simply to use `std::pow` everywhere that you use `pow`.